Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.4
+
+# cache-fill before COPY
+# (docker/docker via git clone --depth=1 for speed)
+RUN mkdir -p /go/src/github.com/docker \
+	&& git clone --depth=1 https://github.com/docker/docker.git /go/src/github.com/docker/docker
+ENV GOPATH $GOPATH:/go/src/github.com/docker/docker/vendor
+# (also, naughty vbatts using things outside pkg/ from docker/docker)
+RUN cd /go/src/github.com/docker/docker \
+	&& bash hack/make/.go-autogen
+RUN go get -d -v github.com/Sirupsen/logrus
```

Comment:
That just makes the `go get` below super fast and no-op so you get shiny fast cache.  Also, if you add more deps later, the `go get` below makes sure they're covered.